src/igvfd/schemas/changelogs/analysis_step.md

```diff
@@ -4,6 +4,8 @@
 
 * Extend `output_content_types` enum list to include `sample barcode count matrix`.
 * Extend `input_content_types` enum list to include `sample barcode count matrix`.
+* Extend `analysis_step_types` enum list to include `computational model training`.
```

Comment:
The newer changelog entries should be at the top (probably just a mix up during rebasing?)

Reply:
yes, fixed